udata/core/dataset/search.py

```diff
@@ -246,7 +249,8 @@ def serialize(cls, dataset):
                 geozones.append({
                     'id': zone.id,
                     'name': zone.name,
-                    'code': zone.code
+                    'keys': [key for key in zone.keys
```

Comment:
I think there is now two path for this with etalab/geozones#4:
- zone.keys is a string
- zone.keys is an array of string